Tests/Datagrid/ProxyQueryTest.php

```diff
@@ -99,10 +100,19 @@ public function testGetFixedQueryBuilder($class, $alias, $id, $expectedId, $valu
         $qb->expects($this->once())
             ->method('setParameter')
             ->with($this->equalTo($expectedId), $this->equalTo(array($value)));
-        $qb->expects($this->once())
+        $qb->expects($this->any())
             ->method('getDQLPart')
-            ->with($this->equalTo('from'))
-            ->willReturn(array(new From($class, $alias)));
+            ->will($this->returnCallBack(function ($part) use ($class, $alias) {
+                $parts = array(
+                    'from' => array(new From($class, $alias)),
+                    'orderBy' => array(new OrderBy('whatever', 'DESC')),
+                );
+
+                return $parts[$part];
+            }));
+        $qb->expects($this->once())
+            ->method('addOrderBy')
+            ->with("$alias.$id", null);
```

Comment:
i donÄt know, in general its not so readable to me, but in this case it might be fine!

Lets keep it